content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/mto.rst

```diff
@@ -2,49 +2,53 @@
 Replenish on order (MTO)
 ========================
 
+.. |SO| replace:: :abbr:`SO (sales order)`
+.. |SOs| replace:: :abbr:`SOs (sales orders)`
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |PO| replace:: :abbr:`PO (purchase order)`
 .. |MTO| replace:: :abbr:`MTO (make to order)`
 .. |RFQ| replace:: :abbr:`RFQ (request for quotation)`
 .. |BOM| replace:: :abbr:`BOM (bill of materials)`
 
 *Replenish on order*, also known as *MTO* (make to order), is a replenishment strategy that creates
```

Comment:
Do you think it's necessary to mention that 'Multi-Step Routes' feature needs to be enabled?